modules/40-define-functions/600-type-annotations/description.en.yml

```diff
@@ -0,0 +1,51 @@
+---
+
+name: Type Annotations
+theory: |
+
+  **Type annotations** are an ability to specify parameter types and return values for functions in Python. This is not a mandatory requirement of the language, but it can help programmers in further development, improve the readability of the code and increase its reliability.
+
+  Let's look at a simple example of a function without type annotations:
+  
+  ```python
+  def concat(first, second):
+      return first + second
+  ```
+  
+  This function concatenates two strings into one, but it's not immediately obvious from looking at the code what types the arguments are and that the function operates on strings rather than adding, for example, two numbers.
```

Comment:
This function concatenates two strings into one. At first glance, it is hard to understand what is going on in the code: what types the arguments have, why the function works with strings and not adding, for example, two numbers.